osgi.bnd

```diff
@@ -21,16 +21,13 @@
 # imports by examining the class files, and generates the OSGi meta-info
 # based on that and this file.
 
--classpath: build/classes
 -failok: false
 
```

Comment:
@kelemen 

I recently had to do with reproducible builds. 
In this context it might be worth considering adding the following bnd properties:

```
-reproducible: true
-noextraheaders: true
```

See [here](https://bnd.bndtools.org/instructions/reproducible.html) and [here](https://bnd.bndtools.org/instructions/noextraheaders.html)

Reply:
Makes sense, but in this PR, my goal is to reproduce the Ant build the best I can. So, I think it would be better to have such changes be done in a separate PR (after merging this). And I guess, then it would be better, if someone more competent in OSGI (I myself have very limited experience and knowledge) did that (I assume you are a lot more competent than me in OSGI).